prime-router/src/main/kotlin/azure/HttpUtilities.kt

```diff
@@ -362,6 +363,19 @@ class HttpUtilities {
                 return responseCode to response
             }
         }
+
+        /**
+         * Mimic String.toLongOrNull()
+         * @return a valid UUID, or null if this [str] cannot be parsed into a valid UUID.
+         */
+        fun toUuidOrNull(str: String): UUID? {
```

Comment:
There is only one use of this code, so I recommend just moving it to where it is used.  Also, the UUID check does not seem to belong in this utility.